modulegen/internal/module/exec.go

```diff
@@ -0,0 +1,19 @@
+package module
```

Comment:
Wdyt having a more specific name, representing the tools that we are going to run:
```suggestion
package tools
```